scripts/source-maintenance.sh

```diff
@@ -68,47 +126,79 @@ while [ $# -ge 1 ]; do
         then
             printUsage
             echo "Error: Option $1 expects an argument."
-            exit 1;
+            exit 1
         fi
         UpdateContributorsSince="$2"
-        shift 2;
+        shift 2
         ;;
     --help|-h)
-        printUsage
-        exit 0;
+        printHelp
+        exit 0
         ;;
     --with-astyle)
         ASTYLE=$2
         shift 2
         ;;
+    --only-changed-since)
+        OnlyChangedSince="$2"
+        shift 2
+        ;;
     *)
         printUsage
         echo "Unsupported command-line option: $1"
-        exit 1;
+        exit 1
```

Comment:
This and similar `;` removals in otherwise untouched code feel out of scope. I do _not_ insist on undoing them, but such changes often make future conflict resolution more difficult.

Reply:
Besides a followup PR to improve MacOS/homebrew support, I would be surprised if anyone were to intensively work on this code soon; I'm inclined to leave this in